Documentation/README_Solver.md

```diff
@@ -1,40 +1,155 @@
+Furthermore, the Solver system deterministically defines an order of operations for these transform calculations as there is no reliable way to specify to Unity the update order for components.
 
-## How to use a solver ##
-First, add the desired behavior to an object by using the respective solver type. All solvers are based on [`Solver.cs`](https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/mrtk_release/Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers/Solver.cs).
+Solvers offer a range of behaviors to attach objects to other objects or systems. One other example would be a tag-along object that hovers in front of the user (based on the camera). A solver could also be attached to a controller and an object to make the object tag-along the controller. All solvers can be safely stacked, for example a tag-along behavior + surface magnetism + momentum.
+
+## How to use a solver
 
 The Solver system consists of three categories of scripts:
 
-* **SolverBase**: The base abstract class that all solvers to derive from. It provides state tracking, smoothing parameters and implementation, automatic solver system integration, and update order.
-* **SolverHandler**: Sets the reference object (eg, the main camera transform) and handles gathering of solver components and executes them in order.
+* [`Solver`](xref:Microsoft.MixedReality.Toolkit.Utilities.Solvers.Solver): The base abstract class that all solvers derive from. It provides state tracking, smoothing parameters and implementation, automatic solver system integration, and update order.
+* [`SolverHandler`](xref:Microsoft.MixedReality.Toolkit.Utilities.Solvers.SolverHandler): Sets the reference object to track against (ex: the main camera transform, hand ray, etc.), handles gathering of solver components, and executes updating them in the proper order.
 
 The third category is the solver itself. The following solvers provide the building blocks for basic behavior:
 
-* **Orbital**: Locks to a specified position * offset from the referenced object.
-* **ConstantViewSize**: Scales to maintain a constant size relative to the view of the referenced object.
-* **RadialView**: Keeps the object within a view cone cast by the referenced object.
-* **SurfaceMagnetism**, casts rays to surfaces in the world, and align the object to that surface.
-* **Momentum**: Applies acceleration/velocity/friction to simulate momentum and springiness for an object being moved by other solvers/components.
-* **InBetween**: Keeps an object in between two tracked objects.
+* [`Orbital`](xref:Microsoft.MixedReality.Toolkit.Utilities.Solvers.Orbital): Locks to a specified position and offset from the referenced object.
```

Comment:
Consider linking to the text sections below, which provide I think more helpful resources for folks getting started in this area. Down below there are xrefs anyway, so nothing is really lost I think, but you gain from pointing to the most helpful getting started/usage docs

Reply:
Good idea